.github/workflows/test.yml

```diff
@@ -2,7 +2,7 @@ name: Test
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
We still use master, let's keep this as is for now.

Reply:
Yes, but everyone's local branches (if they forked and use our repo as upstream) are on `master` and we'd need to make a note somewhere to let everyone know we've switched to `main`.

I'm all for `main`, but I don't think we need to make the switch while we make all these other changes.